src/component/card/card.tsx

```diff
@@ -2,11 +2,23 @@ import type { FC } from 'react'
 import './card.scss'
 import classNames from 'classnames'
 
+export type CardBackground =
+  | 'primary-color'
+  | 'primary-color-variant-1'
+  | 'primary-color-variant-2'
+  | 'primary-color-variant-3'
+  | 'card'
+
 export type CardProps = {
   children: React.ReactElement
+  background?: CardBackground
```

Comment:
For me this property is not needed. We can already add style to the card with the mainClassName property

Reply:
Hmm, for this one we decided to keep all the card background styles outside of interact.scss (basically keep everything in card.scss) as the background images are also stored there 🤔